docs/integrations/sources/microsoft-sharepoint.md

```diff
@@ -27,7 +27,7 @@ This page contains the setup guide and reference information for the Microsoft S
 5. Enter **Drive Name**. To find your drive name go to settings and at the top of setting menu you can find the name of your drive.
 6. Select **Search Scope**. Specifies the location(s) to search for files. Valid options are 'ACCESSIBLE_DRIVES' for all SharePoint drives the user can access, 'SHARED_ITEMS' for shared items the user has access to, and 'ALL' to search both. Default value is 'ALL'.
 7. Enter **Folder Path**. Leave empty to search all folders of the drives. This does not apply to shared items.
-8. The **OAuth2.0** authorization method is selected by default. Click **Authenticate your Microsoft SharePoint account**. Log in and authorize your Microsoft account.
+8. The **OAuth2.0** authorization is selected by default for authentication. You have the option to choose the `grant_type`: `authorization_code`, which uses a refresh token to maintain access without frequent re-authorizations, or `client_credentials`, which does not require a refresh token. After selecting the desired grant type, click **Authenticate your Microsoft SharePoint account**, log in, and authorize your Microsoft account to connect it with Airbyte.
```

Comment:
is it tested manually? 

I think we hide all logic in the UI by one button and all data inputs from DB

Reply:
Yes, I tested it locally.